index.js

```diff
@@ -188,15 +148,6 @@ Client.prototype.sync = function (callback) {
     } catch (ex) {}
   }
```

Comment:
Do we still need this entire `if` block? Do we still need `bootstrap.js` for that matter? Contributions get stuck in a loop in here with no more rulesV2Stamp.

Reply:
Hmm, probably don't need it you're right. Will investigate.

Reply:
Does not appear that it is being used anymore. We can remove this if block + bootstrap, thanks

Reply:
Removed 👍